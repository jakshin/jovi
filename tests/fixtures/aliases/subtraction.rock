say 42 - 1
say 42 minus 2
say 42 without 3
