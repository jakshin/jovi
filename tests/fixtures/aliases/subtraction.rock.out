41
40
39
